x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/index.ts

```diff
@@ -6,13 +6,16 @@
  */
 
 import {
+  EuiMarkdownEditorUiPlugin,
   getDefaultEuiMarkdownParsingPlugins,
   getDefaultEuiMarkdownProcessingPlugins,
+  getDefaultEuiMarkdownUiPlugins,
 } from '@elastic/eui';
 
 import * as timelineMarkdownPlugin from './timeline';
-
-export const uiPlugins = [timelineMarkdownPlugin.plugin];
+const uiPlugins: EuiMarkdownEditorUiPlugin[] = getDefaultEuiMarkdownUiPlugins();
```

Comment:
Clears up 195 instances of the following warning:
```
Deprecation warning: uiPlugins passed to EuiMarkdownEditor does not include the tooltip plugin, which has been added for you. This automatic inclusion has been deprecated and will be removed in the future, see https://github.com/elastic/eui/pull/4383
```
Deprecation, no ones fault but @chandlerprall ;)